ocaml/idl/datamodel_common.ml

```diff
@@ -10,7 +10,9 @@ open Datamodel_roles
               to leave a gap for potential hotfixes needing to increment the schema version.*)
 let schema_major_vsn = 5
 
-let schema_minor_vsn = 774
+(* This comment is a reminder to update the version in upgrade_update_guidance if schema_minor_vsn
+   changed when the update guidance improvement feature branch got merged into master branch *)
+let schema_minor_vsn = 775
```

Comment:
Please add a `let nile_release_schema_major_vsn` and `let nile_release_schema_minor_vsn` at the bottom of the list below, and refer to it from the upgrade rule.